rb_ws/src/buggy/scripts/auton/rolling_sanity_check.py

```diff
@@ -85,17 +90,24 @@ def calc_covariance(self):
                 self.warning_durations[16] = 0
             self.covariance_status_publisher.publish(good_covariance)
 
-
     def calc_locations(self):
-        # currently comparing cross-track error, checking less than 0.5 m
+        # currently comparing distance between rear antenna and filtered loc
         if (self.filter_location == None or self.gps_location == None):
             self.filter_gps_status_publisher.publish(False)
         else:
-            good_seperation = abs(self.filter_location.pose.position.y - self.gps_location.position.y) < 0.5
-            if (not good_seperation):
+            filter_pose = Pose.rospose_to_pose(self.filter_location.pose)
+            filter_loc = World.gps_to_world_pose(filter_pose)
+            rear_antenna_pose = Pose.rospose_to_pose(self.gps_location.pose)
+            rear_antenna_loc = World.gps_to_world_pose(rear_antenna_pose)
+
+            diff = filter_loc - rear_antenna_loc
+            dist = np.sqrt(diff.x ** 2 + diff.y ** 2)
```

Comment:
What happens if the z coordinates start to differ, do we not care?

Reply:
we don't care

Reply:
in fact the z coordinate is thrown out upon conversion to a Pose object